README.md

```diff
@@ -43,12 +43,11 @@
 
 ## 📖 Math Magician <a name="about-project"></a>
 
-
-
-
+**[MATH-MAGICIAN]** is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to Make simple calculations and Read a random math-related quote.
 
 ## 🛠 Built With  VSCode  <a name="Built With VSCode"></a>
 
+
 ### HTML/CSS, javascript, Webpack <a name="tech-stack"></a>
 
 >
```

Comment:
- [ ] You are still missing the tech stark heading for the content below please add it, all sections from the microverse readme should be present in your readme  👍🏿